daemon/containerd/image_list.go

```diff
@@ -257,36 +275,39 @@ func (i *ImageService) Images(ctx context.Context, opts imagetypes.ListOptions)
 	return summaries, nil
 }
 
-func (i *ImageService) singlePlatformImage(ctx context.Context, contentStore content.Store, repoTags []string, imageManifest *ImageManifest, opts imagetypes.ListOptions, allContainers []*container.Container) (*imagetypes.Summary, []digest.Digest, error) {
-	diffIDs, err := imageManifest.RootFS(ctx)
-	if err != nil {
-		return nil, nil, errors.Wrapf(err, "failed to get rootfs of image %s", imageManifest.Name())
-	}
-
+func (i *ImageService) singlePlatformSize(ctx context.Context, imgMfst *ImageManifest) (totalSize int64, contentSize int64, _ error) {
```

Comment:
This name is a bit confusing. I need to look into what exactly it returns;  `totalSize` is the size for all platforms? what is `contentSize` in this context?

Reply:
Oh! This is for a single-arch image (so "total" for the image, in this case being the total for a single arch, correct?)

Reply:
Yes, total here means both content and "unpacked" size.